locales/en-US/webextension.properties

```diff
@@ -27,6 +27,8 @@ loginErrorDetails = We couldn’t save your shot because there is a problem with
 unshootablePageErrorTitle = We can’t screenshot this page.
 unshootablePageErrorDetails = This isn’t a standard Web page, so you can’t take a screenshot of it.
 selfScreenshotErrorTitle = You can’t take a shot of a Firefox Screenshots page!
+# Fired when someone makes a zero-width or zero-height selection
+emptySelectionErrorTitle = Your selection is too small
```

Comment:
Localization happens in GitHub, outside of mozilla-central, so you're not bound to trains: the content that you land in mozilla-central is not exposed to localizers, so no problem with uplifts.

Having said that, the sooner you land, the better. And if you're planning to enable in 55, website's localization is getting kind of late.